rasa/core/tracker_store.py

```diff
@@ -751,6 +751,21 @@ def ensure_schema_exists(session: "Session") -> None:
             raise ValueError(schema_name)
 
 
+def validate_port(port: Any) -> Optional[int]:
+    """Ensure that port can be converted to integer.
+
+    Raises:
+        RasaException if port cannot be cast to integer.
+    """
+    if port and not isinstance(port, int):
```

Comment:
nit: lots of things can evaluate to false (like `if 0.0 and ...` would also evaluate to false), so I think it might be better readable to more explicitly check for `port is not None`